_content/tour/rus/context.article

```diff
@@ -1,25 +1,25 @@
-Context Package
-The package context defines the Context type, which carries deadlines, cancellation signals, and other request-scoped values across API boundaries and between processes.
+Пакет Context
+Пакет context определяет тип `Context`, который переносит сроки, сигналы отмены и другие значения, связанные с запросами, через границы API и между процессами.
 
-* Context Package
+* Пакет Context
 
-- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Watch The Video]]
-- Need Financial Assistance, Use Our [[https://www.ardanlabs.com/scholarship/][Scholarship Form]]
+- [[https://www.ardanlabs.com/training/individual-on-demand/ultimate-go-bundle/][Посмотреть видео]]
+- Если вам нужна финансовая помощь - заполните нашу [[https://www.ardanlabs.com/scholarship/][заявку на стипендию]]
 
-The package context defines the Context type, which carries deadlines, cancellation
-signals, and other request-scoped values across API boundaries and between processes.
+Пакет `context` определяет тип `Context`, который переносит сроки, отмены
+сигналы и другие значения, связанные с запросами, через границы API и между процессами.
 
-** Code Review
+** Обзор кода
 
-"Context values are for request-scoped data that passes through programs in a
-distributed system.
+"Значения контекста предназначены для данных, копируемых по запросу, которые проходят через программы в
```

Comment:
After being stuck on translating this sentence for too long ... Gave it to ChatGPT ...
This word collection has no meaning or is very far from the origin...

HELP Please... :)